pkg/apis/externaldns/types.go

```diff
@@ -64,6 +65,7 @@ func (cfg *Config) ParseFlags(args []string) error {
 	flags.StringVar(&cfg.KubeConfig, "kubeconfig", "", "path to a local kubeconfig file")
 	flags.StringVar(&cfg.Namespace, "namespace", v1.NamespaceAll, "the namespace to look for endpoints; all namespaces by default")
 	flags.StringVar(&cfg.Zone, "zone", "", "the ID of the hosted zone to target")
+	flags.StringVar(&cfg.Domain, "domain", "example.org.", "the name of the top-level domain to manage")
```

Comment:
@ideahitme I think the idea is to "be safe" for as long as ownership is not completely implemented. I agree that the default should empty (work on all domains).